js/main.js

```diff
@@ -44,14 +44,23 @@ document.addEventListener("DOMContentLoaded", function(event) {
         }
       });
 
-      var owl = $('.solstice-slider');
-      owl.owlCarousel({
-          items:1,
-          autoplay:true,
-          autoplayTimeout:6000,
-          autoplayHoverPause:true,
-          autoplaySpeed: 2000,
-          loop:true,
+      var owlSolsticeSlider = $('.solstice-slider');
+      owlSolsticeSlider.owlCarousel();
+
+      const solsticeSliderHome = () => {
+        var owlSolsticeFeaturedStorySlider = $('.solstice-featured-story-slider');
```

Comment:
We don't/shouldn't be using var here.

I still think that the solution that I provided makes more sense but this is better than the previous revision.

$(window).on("load", function() { is fired when the whole page has loaded, including all dependent resources such as stylesheets and images.
 
 
 The solution that I provided can insert the owl carrousel right once we got the data from the API.

Reply:
I tried using the solution you suggested and it didn't seem work. The content is being added to the page but Owl isn't updated to create the carousel.

Reply:
You are right, this looks like an outdated version of the docs.